pkg/services/ngalert/api/testing_api.go

```diff
@@ -29,6 +29,6 @@ func (f *TestingApiHandler) handleRouteEvalQueries(c *models.ReqContext, body ap
 	return f.svc.RouteEvalQueries(c, body)
 }
 
-func (f *TestingApiHandler) handleBacktestingConfig(ctx *models.ReqContext, conf apimodels.BacktestConfig) response.Response {
+func (f *TestingApiHandler) handleBacktestConfig(ctx *models.ReqContext, conf apimodels.BacktestConfig) response.Response {
```

Comment:
Swagger generates the hook spelled like `handleBacktestConfig`, renamed the entry point to match.
Let me know if you'd prefer to go the other way, we can also instead tell Swagger to generate the hook `handleBacktestingConfig`.